Backend/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "src/server.js",
   "scripts": {
     "build": "babel ./src/ -d dist",
-    "start": "npm run build && node ./dist/server.js"
+    "start": "npm run build && node ./dist/server.js",
+    "start-dev": "npm run build && node ./dist/server.js dev"
```

Comment:
Might be best to make dev a flag and then you can call `npm start` or `npm start --dev`

Reply:
That's originally what I had (kind of). You could run `npm start` or `npm start dev`. However, the issue arose with consistency between running these commands in `Backend/` and the top-level directory. I could do it in both easily enough by changing `npm start`'s definition to `"run-p data-generator frontend \"backend"` at the top level. However, that left the data generator running when connecting to the pi (what I hated that I had overlooked). Then, I tried with a flag like you're mentioning (`npm start --dev`), but ran into issues with parsing the flag in different environments ($flag for Linux vs %flag% for Windows). The cross-env tools broke down when I tried running conditional/complex commands in the script. I can give it a try using a separate script to parse the CLAs though. I might save it for the LTE addition though since using flags to specify what connection method to use (before we have a way to automatically switch between them) will be more important then.